### PR TITLE
Add pdfinfo support

### PR DESCRIPTION
**Pull request by giancarlobi**
Width and height could be into flv:pdfinfo or flv:identify, if both null then to default value.

### CODE REVIEW

File: d8content/metadatadisplays/iiif_manifest_2.1.twig.txt [hunk 1]
```diff
@@ -44,7 +44,7 @@ Updated Dec 14th 2020
 {
 "label": "description",
 "value": {{ data.description|json_encode|raw }}
-},
+},pgrade from 19
```
Comment (DiegoPino) [inline d8content/metadatadisplays/iiif_manifest_2.1.twig.txt]: @giancarlobi this seems to be a typo

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Tiny typo/left over comment

File: d8content/metadatadisplays/iiif_manifest_2.1.twig.txt (hunk 1 above)
Comment (giancarlobi) [reply d8content/metadatadisplays/iiif_manifest_2.1.twig.txt]: @DiegoPino Sorry friend, of course, It is, it has to be the same than here https://github.com/esmero/archipelago-deployment/blob/1.0.0-RC1D9/d8content/metadatadisplays/iiif_manifest_2.1.twig.txt. Do I have to change, right? I don't know the better workflow for this, tks

Comment (DiegoPino) [reply d8content/metadatadisplays/iiif_manifest_2.1.twig.txt]: Yes. Edit the file directly but on this branch
https://github.com/esmero/archipelago-deployment/tree/giancarlobi-patch-pdfinfo
Then the pull will be updated automatically

Comment (giancarlobi) [reply d8content/metadatadisplays/iiif_manifest_2.1.twig.txt]: Done, I hope in the right way, friend

**Review by DiegoPino — APPROVED** [review]
Great. Merging! Thanks @giancarlobi

**Pull request merged**